app/controllers/application_controller.rb

```diff
@@ -98,4 +98,15 @@ def after_sign_in_path_for(resource)
     dashboard_path
   end
 
+  #used to deliver notification on various events
+  def notify_users(action, object_type, object_id, victims)
+    Notification.create(
+          :actor => current_user,
+          :action => action, 
+          :object_type => object_type,
+          :object_id => object_id,
+          :victims => victims
+        )    
+  end
```

Comment:
@rohitpaulk May be we can move logic of deciding which action number to use in this function as well? we can pass action argument as strings like "project_comment", "project_follow", "project_fork", "follow_user" etc.
So function call would look something like:
```Ruby
notify_users "project_comment", 1, @comment.id, victims
```